File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        vector<int> res(2);
        vector<int> arr(n);
        for(int i = 0; i < n; i++){
            arr[i]=target - nums[i];
            for(int j = 0; j < n; j++){
                if (arr[i] == nums[j] && i!=j){
                    res[0]=i;
                    res[1]=j;
                    return res;
                }
            }
        }
        
    }
};

File: Data Structures & Algorithms/sort-colors/submission-1.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        r = 0
        w = 0
        b = 0
        for num in nums:
            if num==0:
                r += 1
            elif num == 1:
                w += 1
            else:
                b += 1
        
        arr = [0] * r + [1] * w + [2] * b
        print(arr)

        nums[:] = arr[:]


        